pkg/riff/commands/credentials.go

```diff
@@ -0,0 +1,165 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/projectriff/riff/pkg/core"
+	"github.com/projectriff/riff/pkg/core/tasks"
+	"github.com/projectriff/riff/pkg/env"
+	"github.com/spf13/cobra"
+	"k8s.io/api/core/v1"
+	"strings"
+)
+
+const (
+	credentialsSetNumberOfArgs = iota
+)
+
+const (
+	credentialsListNumberOfArgs = iota
+)
+
+const (
+	credentialsDeleteSecretNameStartIndex = iota
+	credentialsDeleteMinNumberOfArgs
+)
+
+func Credentials() *cobra.Command {
+	return &cobra.Command{
+		Use:   "credentials",
+		Short: "Interact with credentials related resources",
+	}
+}
+
+func CredentialsSet(c *core.Client) *cobra.Command {
+	options := core.SetCredentialsOptions{}
+
+	command := &cobra.Command{
+		Use:     "set",
+		Short:   "create or update secret and bind it to the " + env.Cli.Name + " service account (created if not found)",
```

Comment:
The name of the service account is always riff-build, the name of the cli is irrelevant here